www/docs/guides/compiled_contracts/deployBraavos.ts

```diff
@@ -0,0 +1,197 @@
+// Collection of functions for Braavos account creation
+// coded with Starknet.js v5.11.1, 01/jun/2023
+
+import {
+  BigNumberish,
+  CairoVersion,
+  CallData,
+  Calldata,
+  DeployAccountContractPayload,
+  DeployAccountContractTransaction,
+  DeployContractResponse,
+  EstimateFeeDetails,
+  InvocationsSignerDetails,
+  Provider,
+  RawCalldata,
+  constants,
+  ec,
+  hash,
+  num,
+  stark,
+} from 'starknet';
+
+const BraavosProxyClassHash: BigNumberish =
```

Comment:
Added a commit that changes some type references, e.g. here the change was from `num.BigNumberish` to `BigNumberish`. `num.BigNumberish` will still be supported but direct imports or using it as `types.BigNumberish` should be preferred. This was a change introduced in PR #627 to make the type usage consistent across the library.